packages/cascara/src/ui/ActionsMenu/ActionsMenu.js

```diff
@@ -57,27 +48,14 @@ const ActionsMenu = ({ trigger = DEFAULT_TRIGGER, actions }) => {
           className='menu transition visible'
           style={{ position: 'initial' }}
         >
-          {actions.map(
-            ({ actionName, content, isLabeled, ...rest }, actionIndex) => {
-              // FDS-137: use action name for button name if no content is specified
-              const buttonText = content || rest.name;
-              const key = `action.${actionIndex}-${rest.name}.${content}`;
-
-              return (
-                <MenuItem
-                  {...menu}
-                  {...rest}
-                  as='div'
-                  className={`item ${styles.ActionsMenuItem}`}
-                  key={key}
-                  // eslint-disable-next-line react/jsx-no-bind -- rest needs to be passed, hence the in-situ arrow function
-                  onClick={() => handleMenuItemClick(rest)}
-                >
-                  {buttonText}
-                </MenuItem>
-              );
-            }
-          )}
+          {actions.map((action, actionIndex) => (
+            <MemoActionsMenuItem
+              key={action.name}
+              {...action}
+              actionIndex={actionIndex}
+              menu={menu}
+            />
+          ))}
```

Comment:
I think this is another example we should put together for the team to show them what sorts of things we need to be doing in React now. It is not necessarily that clear from lint rule suggestions what to do in a `.map()`

Reply:
> @lordpixel this all looks great to me. Looks like the build is failing so I will take a look at that now to see why. Great work on this!

The fixture you added breaks one or two eslint rules... among other things:

- No prop types
- arrow functions